app/forms/hyrax/book_contribution_form.rb

```diff
@@ -11,19 +12,16 @@ class BookContributionForm < Hyrax::Forms::WorkForm
 
     self.model_class = ::BookContribution
     self.terms = %i[title resource_type creator alt_title contributor rendering_ids abstract date_published
-                      institution org_unit project_name funder fndr_project_ref series_name book_title editor volume edition  pagination
-                      publisher place_of_publication isbn issn eissn date_accepted date_submitted official_link related_url
-                      language license rights_statement rights_holder doi alternate_identifier related_identifier refereed
-                      keyword dewey library_of_congress_classification add_info]
+                    institution org_unit project_name funder fndr_project_ref series_name book_title editor volume edition
+                    publisher place_of_publication isbn issn eissn date_accepted date_submitted official_link related_url
+                    language license rights_statement rights_holder doi alternate_identifier related_identifier refereed
+                    keyword dewey library_of_congress_classification add_info pagination]
```

Comment:
I'm not sure if this is a safe move or not because I think the order in the terms array sets the order of the fields on the deposit form.